docs/concepts/consensus-protocol/unl.md

```diff
@@ -0,0 +1,36 @@
+
+Initially, it was believed that 60% overlap between two servers' UNLs was enough to prevent those servers from forking apart. However, [further research](./consensus-research.md) showed that in the worst case scenario, 90% overlap was required to prevent a fork. This significantly limits how much flexibility server operators have in customizing their UNL: the less overlap with the UNLs that others are using, the higher the chances of forking.
+
+## Recommended Validator Lists
+
+To make it easier to get a diverse and reliable list of validators that has high overlap with others, the XRP Ledger uses a system of recommended validator lists. A server can be configured to download a recommended list from a _publisher_, and to use that list as its UNL. A server can also be configured with multiple publishers' lists, and to use the union of those lists—meaning the server's UNL consists of every validator that is on _any_ of its published lists. Validators who appear on multiple lists are only included once in the server's UNL.
+
+A recommended list can be identified by the public key of its publisher. Typically, recommended lists are associated with a website where they can be downloaded, but lists can also be relayed through the peer-to-peer network in case there are problems accessing the website.
+
+Currently, the default configuration for XRP Ledger servers uses two lists: one published by the XRP Ledger Foundation, and one published by Ripple. Typically, these lists are very similar to one another or even identical. The term _default UNL_ (sometimes abbreviated dUNL) refers to the set of validators included in these lists.
+
+Anyone can publish a signed list of validators in the correct format, which is a JSON document containing signed binary data. For more information on the format of a recommended list, see [Validator List Method](/docs/references/http-websocket-apis/peer-port-methods/validator-list/).
+
+Recommended validator lists need to be updated over time, to add more quality validators and remove validators that aren't as reliable or are retiring. Typically, a recommended validator list has an expiration time, because the publisher expects to have made an update available by that time. Lists also have a sequence number, such that the highest sequence number is the newest version of that list and supercedes any older versions. Lists can also have an activation date so that servers can coordinate when to switch to the new version, and the updated list has time to propagate to all servers that use it.
+
+Publishers aren't involved in day-to-day validation of new transactions, but they do wield significant power in selecting which validators are widely trusted. Server operators should be cautious in selecting which validator list publishers they trust, because carelessness on the part of the publishers could affect the reliability of the servers using those lists.
```

Comment:
So people can technically pick any publisher, but really we really just want them using the two widely accepted ones, right? Maybe it's a good idea here to gently push opinion to using these two lists, and elaborate on why these 2 publishers are so trusted in the space?

Reply:
I don't really want to editorialize more than this section already has, and right now I don't know of any publishers other than the two default ones anyway. (Coil used to run a list as well, but they went out of business.)